app/assets/stylesheets/components.scss

```diff
@@ -1703,7 +1703,7 @@ button.active i.fa-retweet {
 
 .privacy-dropdown__option__content {
   flex: 1 1 auto;
-  color: $color3;
+  color: #536f96;
```

Comment:
This could be `darken($color3, N%)` where N is how many percent darker your version of the color is. The code doesn't have explicit color definitions anywhere but in variables.scss